source/plot/SvgPlot.ooc

```diff
@@ -34,7 +34,10 @@ SvgPlot: class {
 		this datasets free()
 		this xAxis free()
 		this yAxis free()
-		this colorList free()
+		if (this colorList) {
+			this colorList _count = 0
```

Comment:
`colorList` entries are shared (as in the PR description), this is a way around that which allows to free the `colorList` anyway.